src/Smalot/PdfParser/Font.php

```diff
@@ -57,6 +57,9 @@ class Font extends PDFObject
      */
     private static $uchrCache = [];
 
+    /** @var Encoding */
+    private $initializedEncodingByPdfObject;
```

Comment:
```suggestion
    private Encoding $initializedEncodingByPdfObject;
```

Reply:
PHP supports type definitions for class properties just from [version 7.4](https://www.php.net/manual/en/migration74.new-features.php). But currently this project must support php strating from version 7.1 (by composer restriction ("php": ">=7.1")).

And at the moment project hasn't any typed properties.

Therefore I think It would be better keep it as is.

And maybe add issue with task to migrate to php v7.4 and add propetries types everywere in sources.

Reply:
Ah yes, my bad.